dev/breeze/src/airflow_breeze/commands/sbom_commands.py

```diff
@@ -627,3 +634,85 @@ def generate_providers_requirements(
                 force=force,
                 output=None,
             )
+
+
+@sbom.command(name="export-dependency-information", help="Export dependency information from SBOM.")
+@option_airflow_version
+@option_python
+@click.option(
+    "-f",
+    "--csv-file",
+    type=click.Path(file_okay=True, dir_okay=False, path_type=Path, writable=True),
+    help="CSV file to produce.",
+    envvar="CSV_FILE",
+    required=True,
+)
+@click.option(
+    "-s",
+    "--include-open-psf-scorecard",
+    is_flag=True,
+    default=False,
+)
+@option_dry_run
+@option_answer
+def export_dependency_information(
+    python: str,
+    airflow_version: str,
+    csv_file: Path,
+    include_open_psf_scorecard: bool,
+):
+    import requests
+
+    sbom_core_url = (
+        f"https://airflow.apache.org/docs/apache-airflow/{airflow_version}/"
+        f"sbom/apache-airflow-sbom-{airflow_version}-python{python}-python-only.json"
+    )
+    sbom_full_url = (
+        f"https://airflow.apache.org/docs/apache-airflow/{airflow_version}/"
+        f"sbom/apache-airflow-sbom-{airflow_version}-python{python}-python-only-full.json"
+    )
```

Comment:
Could you create a variable for the common part?

Reply:
Absolutely. Done :)